### ISSUE TITLE
1.5 not threadsafe anymore

### ISSUE THREAD

**Issue by djamadeus** [tglunde/dbt-exasol#101]
with 1.4.6 dbt run worked perfectly fine with Threads: 4 in profile
after Upgrade the build randomly fails with errors like this:
```
Internal Error
14:59:06    Cache inconsistency detected: in rename, new key _ReferenceKey(database=None, schema=****', identifier='****') already in cache:
...
```
when lowering threads to 1 in profile.yml everything works fine again.

**Comment by tglunde** [comment]
Hi @djamadeus ,

thanks, that is unfortunately a bummer. Will try to work on this asap - unfortunately I am only back from holiday on October 2nd. @ilikutle, will you have time to look into this?

Regards, Torsten

**Comment by ilikutle** [comment]
Hello @djamadeus, can you please share some more information on the reproduced model. Which materialization is used on the model that throws this error. Or if you could share log file, it would be great.
Regards!

**Comment by djamadeus** [comment]
Hi,
I'd really need to redact those logs a little before handing them out, that might take me a while, or reproduce it in a smaller steup... Will be back on this soon

**Comment by djamadeus** [comment]
Mailed ya

**Comment by ilikutle** [comment]
Hello @djamadeus, we have done some changes in order to deal better with caching, you can check it out in this PR #103.
Let us know if the problem is still there :)

**Comment by tglunde** [comment]
published with 1.5.1
